html/changelogs/geeves-cultstuff.yml

```
@@ -0,0 +1,24 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - bugfix: "Fixed some cult item descriptions not having a full stop"
```

```suggestion
  - bugfix: "Fixed some cult item descriptions not having a full stop."
```
ironic, he could save others from the lack of full stops, but not himself

---

that's the point u shit